### ISSUE TITLE
Arcpy needs to run within active ArcGIS Conda Environment

### ISSUE THREAD

**Issue by cyrusblankinship** [Esri/arcgis-python-api#72]
My issue started the same as  #10, where it was saying 

> _Widget Javascript not detected. It may not be installed properly. Did you enable the widgetsnbextension? If not, then run "jupyter nbextension enable --py --sys-prefix widgetsnbextension_

I uninstalled and reinstalled both packages 

```
conda uninstall -c conda-forge ipywidgets
conda uninstall -c esri arcgis

conda install -c conda-forge ipywidgets
conda install -c esri arcgis
jupyter nbextension enable --py --sys-prefix widgetsnbextension
```

but that made everything worse. I am now getting an error saying  

> _arcpy must be run from inside the Conda environment, which ArcGIS
uses to manage the installation of Python.
You can access this environment by launching Python from the links inside the
"ArcGIS > ArcGIS Pro" Start Menu Group:
    "Python Interactive Terminal" starts a Python session.
    "Python Command Prompt" opens a Command Prompt window initialized with Conda._

**Comment by rohitgeo** [comment]
It appears that you've installed arcgis package in arcgispro-py3 environment using proenv.bat/Python Command Prompt window within ArcGIS Pro, but the notebook server has been started in a different conda environment.

Can you try restarting Jupyter notebook from within the  proenv.bat/Python Command Prompt window in ArcGIS Pro?

**Comment by scw** [comment]
Hello @cyrusblankinship,

Unfortunately the instructions in the other thread are out of date and using `conda-forge` is known to cause some issues. We've resolved this problem for the next release of Pro, but that doesn't help you now. A couple of options:

1. If you're comfortable editing code, open the file:

`<Pro installation directory>\Resources\ArcPy\arcpy\__init__.py`

and change line 31, which currently reads:

```python
if not "Continuum" in sys.version:
```
to instead read:
```python
if not "Continuum" in sys.version and not "conda-forge" in sys.version:
```
That will allow "conda-forge" to pass muster as a provider of a Python installation. (We added this check to prevent some complicated cases that will cause crashes with mixing Python installs).

2. The less technical solution is to entirely delete Pro, remove any remaining directories and files in the `<Pro install>\bin\Python` directory (which will have been modified by installing via `conda-forge`), and then installing Pro again. Finally, then use the instructions [provided in the documentation](https://developers.arcgis.com/python/guide/install-and-set-up/) which should boil down to opening `proenv.bat` and then running:

`conda install -c esri arcgis`

From the prompt.
